src/validator/DtoValidator.java

```diff
@@ -0,0 +1,105 @@
+package validator;
+
+import annotation.*;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class DtoValidator {
+    private static final String REGEX = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";
+
+    public static List<String> validateDto(Object dto) throws IllegalAccessException {
```

Comment:
Instead of using Object, try to make this class Generic

Reply:
Done